native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java

```diff
@@ -0,0 +1,204 @@
+     * Method to update the Config.toml with database configurations.
+     */
+    public static SyntaxTree updateToml(Path configPath, String name) throws IOException {
+
+        boolean isTableEntry = false;
+        ArrayList<String> existingNodes = new ArrayList<String>();
+        NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        Path fileNamePath = configPath.getFileName();
+        TextDocument configDocument = TextDocuments.from(Files.readString(configPath));
+        SyntaxTree syntaxTree = SyntaxTree.from(configDocument, fileNamePath.toString());
+        DocumentNode rootNote = (DocumentNode) syntaxTree.rootNode();
+        NodeList nodeList = rootNote.members();
+
+        for (Object member : nodeList) {
+            if (member instanceof KeyValueNode) {
+                KeyValueNode node = (KeyValueNode) member;
+                if (!isTableEntry) {
+                    if (!isDatabaseConfigurationEntry(node.identifier())) {
+                        moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                    } else {
+                        if (node.identifier().toString().trim().equals(KEY_PORT)) {
+                            existingNodes.add(node.identifier().toString().trim());
+                            moduleMembers = moduleMembers.add(SampleNodeGenerator.createNumericKV(
+                                    node.identifier().toString().trim(),
+                                    defaultValues[indexOf(nodeMap,
+                                            node.identifier().toString().trim())], null));
+                        } else {
+                            existingNodes.add(node.identifier().toString().trim());
+                            moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(
+                                    node.identifier().toString().trim(),
+                                    defaultValues[indexOf(nodeMap,
+                                            node.identifier().toString().trim())], null));
+                        }
+                    }
+                } else {
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                }
+            } else if (member instanceof TableNode || member instanceof TableArrayNode) {
+                if (!existingNodes.isEmpty() || !moduleMembers.isEmpty()) {
+                    moduleMembers = addNewLine(moduleMembers, 2);
+                    moduleMembers = poulateRemaining(moduleMembers, existingNodes);
+                    moduleMembers = addNewLine(moduleMembers, 1);
+                    isTableEntry = true;
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                } else {
+                    isTableEntry = true;
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                }
```

Comment:
Seems this `else` block is redundant?

Reply:
Removed in [e4c809d](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4c809da564408437e379bbe491d495ca2cbd2e8)